tools/console_version

```diff
@@ -1,2 +1,2 @@
-COMMIT="9d3075fc7c8f2ced05a00732903a1ba96e9e209b"
-SHA2="28240cd6c218d22aca93b63e22412944cae7aadbfb809feba9877d252ee4e199"
+COMMIT="a6c1c20dda3acc71c6325392637c54012e157230"
+SHA2="e75c9e2f94cce18bae37d202f2ee422c7e60d30197862d0ca38f4a2cc2686217"
```

Comment:
The CI seems convinced this should be `ec101722f433f8177f07e7ca263a4480b07b37dd3f20201c143fa41fb691c70a` - can we double-check?

Reply:
That's very possible. I generated the hash manually so I could've gotten that incorrect.